tools/Validate-SignedNupkg.ps1

```diff
@@ -0,0 +1,56 @@
+<#
+.SYNOPSIS
+   Validates signed files. Use this code for manual validation of a folder of nupkg files.
+.DESCRIPTION
+   Recursively validates folder for signed files.
+.NOTES
+   Use this script to check files for valid signatures before release.
+#>
+
+param
+(
+    [Parameter(Mandatory=$true)]
+    # $NupkgPath is the path to the folder of .nupkg files.
+    [string] $NupkgPath,
```

Comment:
singlenuget package path or multiple nuget files? the parameter name implies only one